components/Header.jsx

```diff
@@ -12,43 +11,87 @@ import {
+            <li>329.4131.136</li>
+          </ul>
+        </div>
 
-      <nav>
-        <div className={styles.CentralBar}>
-          <div className={styles.menuIcon}>
-            {/* <FontAwesomeIcon icon={faBars} /> */}
-          </div>
-          <img src="/logo.jpg" alt="LogoSHE" />
-          <div className={styles.rightSide}>
-            <div>
-              {/* <FontAwesomeIcon icon={faUserCircle} />{" "} */}
+        <nav className="">
+          <div className="flex items-center justify-between p-3 w-full sm:h-24 lg:h-28">
+            <div className="flex items-center">
+              <div className="sm:hidden">
+                <FontAwesomeIcon icon={faBars} className="h-5 sm:h-7" />
+              </div>
             </div>
-            <div>
-              {/* <FontAwesomeIcon icon={faCartShopping} /> */}
+            <img
+              src="/logo.jpg"
+              alt="LogoSHE"
+              className="h-16 sm:h-19 lg:h-21 sm:order-first"
+            />
+
+            <div className="flex items-center space-x-2">
+              <div className="hidden sm:flex w-72 max-w-[260px] border border-gray-300">
+                <div className="flex items-center text-xs justify-between pl-2 p-1 sm:text-md lg:text-xl w-full border-r border-gray-300 sm:visible">
+                  RICERCA
+                  <FontAwesomeIcon
+                    icon={faMagnifyingGlass}
+                    className="h-4 sm:h-5 ml-1 mr-2"
+                  />
+                </div>
+              </div>
+
+              <div>
+                <FontAwesomeIcon
+                  icon={faUserCircle}
+                  className="h-5 sm:h-6"
+                />
+              </div>
+              <div className="relative flex items-center">
+                <FontAwesomeIcon
+                  icon={faCartShopping}
+                  className="ml-1 h-5 sm:h-6"
+                />
+                <div className="absolute -top-2/3 -right-2 flex items-center rounded-full w-4 h-4 bg-[#5f9b96] text-white justify-center text-xs sm:text-base lg:text-lg">
+                  0
+                </div>
+              </div>
             </div>
           </div>
-        </div>
 
-        <div className={styles.bottonBar}>
-          <div className={styles.searchBar}>
-            {/* CERCARE <FontAwesomeIcon icon={faMagnifyingGlass} /> */}
-          </div>
-          <div className={styles.productDropdown}>
-            {/* PRODOTTI <FontAwesomeIcon icon={faChevronDown} /> */}
-          </div>
-        </div>
-      </nav>
+          <ul className="flex items-center text-sm border-t border border-gray-300 sm:h-10">
+            <li className="sm:hidden flex items-center text-xs justify-between w-3/5 pl-4 p-2 border-r border-gray-300 sm:text-lg lg:text-xl">
+              RICERCA
+            </li>
+
+            <li className="flex items-center text-xs justify-center p-2 w-auto sm:text-lg lg:text-xl">
+              PRODOTTI
+            </li>
```

Comment:
instead of using `flex items-center justify-center` to center text just do `text-center`